lib/v2/datasource/local/util/seeds_esr.dart

```diff
@@ -0,0 +1,61 @@
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:dart_esr/dart_esr.dart';
+import 'package:seeds/v2/datasource/local/models/scan_qr_code_result_data.dart';
+import 'package:async/async.dart';
+
+class SeedsESR {
+  late SigningRequestManager manager;
+
+  late List<Action> actions;
+
+  SeedsESR({String? uri}) {
+    manager = TelosSigningManager.from(uri);
+  }
+
+  Future<void> resolve({String? account}) async {
+    actions = await manager.fetchActions(account: account);
+  }
+
```

Comment:
The code below, with the big comment, was taken from the old code. 

So this is not making the codebase worse, just moved this method here.

It will need to be fixed and done an entirely different way in another PR